Teamleader.php

```diff
@@ -414,6 +414,28 @@ public function crmUpdateContact(
     }
 
     /**
+     * Delete a contact
+     *
+     * @todo    N/A
+     *
+     * @param         $contact can be either an object of type "Contact" or a contact ID
```

Comment:
I would say `@param int|Contact` to show that it can either be an integer (contact id) or a Contact object.